src/integration/data/upsertAndUpdate.test.ts

```diff
@@ -73,7 +73,7 @@ describe('upsert and update', () => {
       values: newValues,
       metadata: newMetadata,
     });
-    await sleep(3000);
+    await sleep(7000);
```

Comment:
Update was causing issues in edge but not necessarily node. I mentioned this in the PR summary, but I'd like to look into a helper function that lets us retry for asserts in a consistent way.